src/v2/components/AddBlock/components/AddBlockPasteUploader/index.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { useState, useCallback } from 'react'
+import styled from 'styled-components'
+
+import Box from 'v2/components/UI/Box'
+import usePasteListener from 'v2/hooks/usePasteListener'
+import FileUploader from 'v2/components/UI/FileUploader'
+
+const DropZone = styled(Box)`
+  ${props => `display: ${{ resting: 'none', active: 'block' }[props.mode]};`}
+  position: fixed;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  z-index: 2;
+`
+
+const Backdrop = styled(Box).attrs({
+  bg: 'utility.translucent',
+})`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+  position: fixed;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  z-index: 3;
+`
+
+interface AddBlockPasteUploader {
+  onAddBlock: (props: any) => any
+  channelId: string | number
+  createBlock: (props: any) => Promise<any>
```

Comment:
Would avoid all these anys. I know there's a lot of them but just get rid of them. Hover to find out return types at the call sites and just paste em around.

Reply:
Now I remember why I did that:
![Screen Shot 2019-10-07 at 1 34 40 PM](https://user-images.githubusercontent.com/821469/66334522-397c4880-e907-11e9-9b15-22c3d400e87a.png)

Will fix.

Reply:
Yeah, I suspect most of my TypeScript from this era isn't exactly exemplar.